libraries/botbuilder-dialogs-adaptive-runtime-integration-express/package.json

```diff
@@ -28,6 +28,7 @@
   ],
   "dependencies": {
     "@types/express": "4.17.1",
+    "body-parser": "^1.19.0",
```

Comment:
Rather than attempting to internalize HTTP request parsing, I decided to push that down to the integration layer. The adapter now expects that `req.body` is parsed. It throws a useful error if that isn't the case, as well.